packages/browsers/src/httpUtil.ts

```diff
@@ -46,22 +46,25 @@ export function httpRequest(
     agent: new ProxyAgent(),
   };
 
-  const requestCallback = (res: http.IncomingMessage): void => {
+  const request =
+    options.protocol === 'https:'
+      ? https.request(options, requestCallback)
+      : http.request(options, requestCallback);
+
+  function requestCallback(res: http.IncomingMessage) {
     if (
       res.statusCode &&
       res.statusCode >= 300 &&
       res.statusCode < 400 &&
       res.headers.location
     ) {
       httpRequest(new URL(res.headers.location), method, response);
+      // Destroy the request to allow the connection to close
+      request.destroy();
```

Comment:
Shouldn't we do `request.end`?

Reply:
No it should be neither. I think we need to do something to the response not the request